.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
+install: true
+
+jdk:
+  - oraclejdk8
```

Comment:
Just a question. Shouldn't we use openjdk instead of oraclejdk?

Reply:
you're right. Thanks for the hint!